ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17-periodics.yaml

```diff
@@ -7013,16 +7101,6 @@ periodics:
     job-release: "4.17"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-aws-ipi-outpost-day2-f9-longduration-cloud
-  reporter_config:
-    slack:
-      channel: '#forum-prow-long-duration-cloudinfra-testcases'
```

Comment:
Unexpected removal ?

Reply:
`make update` does.

Reply:
Need I add it manually ?

Reply:
They are added in https://github.com/openshift/release/pull/54391. 
Maybe fetch latest codes from upstream then rebase ?

Reply:
Yes, fetch latest codes from upstream then rebase

Reply:
Recovered.